crt_portal/static/sass/custom/progress.scss

```diff
@@ -64,17 +64,21 @@ ol.steps {
     display: block;
     text-decoration: none;
     color: white;
+
+    a {
+      display: block;
+    }
   }
 
-  .step::before,
-  .step::after {
+  .step a::before,
+  .step a::after {
```

Comment:
changing these rules to target the `a` instead of the `li.step` was all it took to make the numbers clickable also!

Reply:
Nice!!!